src/Core/src/Platform/Android/BorderDrawable.cs

```diff
@@ -14,8 +18,10 @@
 
 namespace Microsoft.Maui.Platform
 {
-	public class BorderDrawable : PaintDrawable
+	public class BorderDrawable : PaintDrawable, IAnimatorUpdateListener
```

Comment:
We can't add a new interface

Reply:
With `IAnimatorUpdateListener`, should go to .NET 9?. In this case, we need an alternative for .NET 8.